api/api.md

```diff
@@ -42,15 +42,25 @@ Example usage to search users by employeeID:
 https://`siteaddress`/external/api/rest/v2/users?search={employeeID}
 
 ## Resource Requests
-NOTE: When creating/updating an object, all that is needed is the ID of new object
+
+
+NOTE: When updating a resource (with a `PUT` request), all that is needed is the ID of the object, and the value(s) you wish to update.
         
-    e.g. Updating a resource's contact to another with an ID of 5
-    + Request (application/json)
-        {
-            contact: {
-                id: 5
-            }
-        }
+* e.g. Updating a resource's `printerName` to another, when that resource's `id` is `5`.
+
+Headers
```

Comment:
Perhaps give a sample url, usually the endpoints require the id in the url as well. Some, not all.

Reply:
@jmlopez-rod when this section is compiled, a url example is shown. Does this suffice? 
![image](https://user-images.githubusercontent.com/15133199/51490743-d430f900-1d71-11e9-8578-8760d5694241.png)

Reply:
Yes, its enough